lib/modules/auth/signin.dart

```diff
@@ -0,0 +1,209 @@
+                          });
+                        },
+                      ),
+                      SizedBox(
+                          height: MediaQuery.of(context).size.height * 0.08),
+                      Padding(
+                        padding: EdgeInsets.symmetric(
+                            horizontal:
+                                MediaQuery.of(context).size.width * 0.05),
+                        child: CustomButton(
+                          text: AppStrings.signIn,
+                          onPressed: () {
+                            if (signinFormKey.currentState!.validate()) {
+                              ScaffoldMessenger.of(context).showSnackBar(
+                                const SnackBar(
+                                  content: Text('Hey! There '),
+                                  backgroundColor: AppColors.inputLabelColor,
+                                ),
+                              );
+                            }
+                          },
+                        ),
+                      ),
+                      SizedBox(
+                          height: MediaQuery.of(context).size.height * 0.05),
+                      Padding(
+                        padding: EdgeInsets.symmetric(
+                            horizontal:
+                                MediaQuery.of(context).size.width * 0.05),
+                        child: Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                          children: [
+                            Row(
+                              children: <Widget>[
+                                Checkbox(
+                                  shape: RoundedRectangleBorder(
+                                    borderRadius: BorderRadius.circular(2.0),
+                                  ),
+                                  side: MaterialStateBorderSide.resolveWith(
+                                    (states) => const BorderSide(
+                                        width: 1.0,
+                                        color: AppColors.lightYellow),
+                                  ),
+                                  value: rememberMe,
+                                  activeColor: Colors.transparent,
+                                  onChanged: (bool? value) {
+                                    setState(() {
+                                      rememberMe = value!;
+                                    });
+                                  },
+                                ),
+                                const Text(
+                                  AppStrings.rememberMe,
+                                  style: AppTheme.displayMedium,
+                                ),
+                              ],
+                            ),
+                            InkWell(
+                              onTap: () {
+                                ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
You should create a common way of accessing this.

Reply:
Resolved